src/entity/user/User.ts

```diff
@@ -174,6 +179,17 @@ export class User {
   @Column({ type: 'text', nullable: true })
   experienceLevel: string | null;
 
+  // @Column({ type: 'jsonb', default: { score: 1, banned: false } })
+  // @Index('IDX_user_flags_banned', { synchronize: false })
+  // flags: UserFlags;
```

Comment:
Why not flags?

Reply:
It was easier to do without flags as in the CDC, the flags are returned as a JSON string. But I guess if I move the worker to a db trigger, we can use flags instead.

Reply:
Refactored to use flags